Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1755,7 +1755,7 @@
 "PROFILE_VIEW_PROFILE_AVATAR_FIELD" = "Avatar";
 
 /* Description of the user profile. */
-"PROFILE_VIEW_PROFILE_DESCRIPTION" = "Your Signal Profile will be visible to your contacts, when you initiate new conversations, and when you explicitly make it visible to other users and groups.";
+"PROFILE_VIEW_PROFILE_DESCRIPTION" = "Your Signal Profile will be visible only to people who are in your contact list, to people who you intitiated a conversation with, and to people for who you explicitly gave permission. Signal profiles are end-to-end encrypted, so even the Signal service never has access to your profile information.";
```

Comment:
"and to people for who you explicitly gave permission"

I think this should read "and to people for whom you explicitly gave permission".

Reply:
In Android you give your app permission for this action of making your profile visible to someone by clicking on a banner "share my profile with this person".
I am not sure if that also happens in iOS. In what way can you share your profile without initializing a conversation ans without adding a person as a contact? Can you help by showing me a screenshot of that?

Reply:
Sorry, no idea. Also, I don't have access to an iOS device.

Reply:
I found the info, will change accordingly: https://signal.org/blog/signal-profiles-beta/

Reply:
Still, it should actually be "whom" instead of "who" but I am not sure if that might be too formal.